cfme/base/credential.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+from cfme.web_ui import FileInput, Input, Radio, form_buttons
+from cfme.web_ui.tabstrip import TabStripForm
+from utils import conf
+from utils.pretty import Pretty
+from utils.update import Updateable
+from widgetastic.utils import Fillable
+
+
+class FromConfigMixin(object):
+    @classmethod
+    def from_config(cls, key):
+        creds = conf.credentials[key]
+
+        # todo: fix this along with other credential fixes. code or credentials conf ?
+        to_rename = [('password', 'secret'), ('username', 'principal')]
```

Comment:
honestly, I haven't decided how to get rig of this temporary fix. Possibly, I'll make some mapping inside class and move this mapping to class attribute. I'd like to also check whether it makes sense to rename appropriate attributes in credentials file.